task/views.py

```diff
@@ -50,18 +52,33 @@ def get(self, request, id=None):
         task = Task.objects.filter(user__id=request.user.id)
         serializer = self.serializer_class(task, many=True)
         return Response(serializer.data)    
-
     
 class CheckTaskView(APIView):
     serializer_class = TaskSerializer
     
     def get(self, request, id=None):
-        task = Task.objects.filter(status='Todo')
+        task = Task.objects.all()
         page_number = request.GET.get('page_number', 1)
         page_size = request.GET.get('page_size', 50)
         paginator = Paginator(task, page_size)
         serializer = self.serializer_class(paginator.page(page_number), many=True)
         return Response(serializer.data)
-    
    
-                
\ No newline at end of file
+                
+class UserstatusView(APIView):
```

Comment:
Doesn't meant to be pushed. Just for tests.

Reply:
fixed